frontend/public/actions/features.ts

```diff
@@ -207,18 +212,36 @@ const detectCanCreateProject = (dispatch) =>
     },
   );
 
-const detectUser = (dispatch) =>
-  // FIXME: use k8s self-subject review API
-  fetchURL('/apis/user.openshift.io/v1/users/~').then(
-    (user) => {
-      dispatch(setUser(user));
-    },
-    (err) => {
-      if (!_.includes([401, 403, 404, 500], err?.response?.status)) {
-        setTimeout(() => detectUser(dispatch), 15000);
-      }
-    },
-  );
+const detectUser = (dispatch: Dispatch) =>
+  client
```

Comment:
I'd avoid using GraphQL unless it's really needed to keep things simple (although I understand you were having issues with fetch?)